whiskey.yml

```diff
@@ -14,7 +19,7 @@ Build:
 - Exec:
       OnlyBy: BuildServer
       Path: appveyor
-      Argument: [ UpdateBuild, -Version, $(WHISKEY_SEMVER2) ]
+      Argument: [ UpdateBuild, -Version, "$(WHISKEY_SEMVER2)+$(WHISKEY_BUILD_NUMBER)" ]
 
 # Dot-sourcing files is expensive. Move all functions into .psm1 file to improve import speed. Do this before testing

```

Comment:
Add this task after the existing `MergeFile` task:

```powershell
- CopyFile:
    Path:
    - LICENSE
    - NOTICE
    - README.md
    - CHANGELOG.md
    DestinationDirectory: RundeckAutomation
```

Reply:
Which means you'll want to add these lines to your .gitignore, too:

```
/RundeckAutomation/CHANGELOG.md
/RundeckAutomation/LICENSE
/RundeckAutomation/NOTICE
/RundeckAutomation/README.md
```

Reply:
Updated whiskey and the file locations.